src/utils/debug/texture-selector-utils.ts

```diff
@@ -42,14 +66,32 @@ export function selectDebugTextureForTile(tile, uvDebugTexture) {
       tile.userData.originalTexture =
         material.pbrMetallicRoughness.baseColorTexture.texture.source.image;
     }
+    const width =
+      material.pbrMetallicRoughness.baseColorTexture.texture.source.image.width;
+    const height =
+      material.pbrMetallicRoughness.baseColorTexture.texture.source.image
+        .height;
```

Comment:
That should work and looks much better
```suggestion
    const width = tile.userData.originalTexture.width;
    const height = tile.userData.originalTexture.height;
```

Reply:
Done